### ISSUE TITLE
match/case statements don't indent properly

### ISSUE THREAD

**Issue by solson** [nixeagle/ooc-mode#5]
Expected indentation:

```
match foo {
    case 12 =>
        "twelve" println()
    case 13 =>
        "thirteen" println()
    case =>
        "who knows?" println()
}
```

ooc-mode's indentation:

```
match foo {
case 12 =>
"twelve" println()
case 13 =>
"thirteen" println()
case =>
"who knows?" println()
}
```

**Comment by nixeagle** [comment]
Match acts like an "if" syntactically from the looks of this?

**Comment by solson** [comment]
Yeah it's pretty much an if, else if, else if, else thing.

**Comment by nixeagle** [comment]
Closed by bada3883e57118fbd187dad9f29a9676781f1753 match/case statements properly indent now.
